Support --input flag to read targets from file

mihails-strasuns-sociomantic opened the pull request:
Trying to pass unknown amount of fully qualified paths as CLI
arguments may hit shell char length limit eventually. Using file
to temporarily store the list is more reliable.

Will be used in updated makd integration for `make d2conv`

mihails-strasuns-sociomantic commented:
I have split this change from dsymbol PR as it is completely independent.

leandro-lucarella-sociomantic commented on src/d1to2fix/main.d:
```diff
@@ -102,14 +102,32 @@ int main ( string[] args )
+    }
+    else
+        fileNames = args[1 .. $];
+
```
Mmm, so is either `d1to2fix @file` or `d1to2fix file1 ... fileN` but `d1to2fix @files1 file1` can't be used? In that case (or in any case) why not using a command line option like -f (or even stdin) to get multiple files? I always found `@file` super weird...

mihails-strasuns-sociomantic commented:
Updated to use dedicated `--input` flag

leandro-lucarella-sociomantic commented on .gitignore:
```diff
@@ -1,3 +1,3 @@
 build/
 deb/*.deb
-.converted
+tests/*.converted
```
Mmm, did this location really changed in this commit or it was an unrelated "bug fix"?

mihails-strasuns-sociomantic replied:
Bug fix, I'll split

mihails-strasuns-sociomantic committed:
Fix generated snippet mask in gitignore

mihails-strasuns-sociomantic committed:
Support --input flag to read targets from file

Trying to pass unknown amount of fully qualified paths as CLI
arguments may hit shell char length limit eventually. Using file
to temporarily store the list is more reliable.

mihails-strasuns-sociomantic commented:
Anything else?
